recipe/build.sh

```diff
@@ -1,28 +1,54 @@
-  sed -i.bak "s|@ARCH@|x64|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  sed -i.bak "s|@OS@|macos|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  sed -i.bak "s|@VERTAIL@|dbru|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  aver=$(echo ${PKG_VERSION} | tr -d ".")
-  sed -i.bak "s|@AVER@|${aver}|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
+CROOT=/opt/conda
 
-elif [[ ${target_platform} == "linux-64" ]]; then
-  sed -i.bak "s|@PLATFORM@|linux|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  sed -i.bak "s|@ARCH@|x64|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  sed -i.bak "s|@OS@|linux|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  sed -i.bak "s|@VERTAIL@||g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
-  # the folder name for the version 23.7.0.25.1 is different than for the previous releases
-  aver=$(echo ${PKG_VERSION:0:6}.00.0 | tr -d ".")
-  sed -i.bak "s|@AVER@|${aver}|g" "${PREFIX}/etc/conda/${CHANGE}.d/${PKG_NAME}_${CHANGE}.sh"
+set -ex
+
+# Create directory for configuration
+mkdir -p $PREFIX/lib
+
+MAJOR_VERSION=$(echo $PKG_VERSION | cut -d. -f1)
+MINOR_VERSION=$(echo $PKG_VERSION | cut -d. -f2)
+PATCH_VERSION=$(echo $PKG_VERSION | cut -d. -f3)
+SUBPATCH_VERSION=$(echo $PKG_VERSION | cut -d. -f4)
+
+OIC_FOLDER_NAME=instantclient_${MAJOR_VERSION}_${MINOR_VERSION}
+
+if [[ ${target_platform} == linux-* ]]; then
+    cp -r ${OIC_FOLDER_NAME}/* $PREFIX/lib/
+fi
+#if [[ ${target_platform} == linux-64 ]]; then
+#    # Extract the .zip
+#    mkdir -p tmp_extract
+#    unzip instantclient-basic-linux.x64-$PKG_VERSION.zip -d tmp_extract/
+#
+#    # Copy contents to target lib folder
+#    cp -r tmp_extract/${OIC_FOLDER_NAME}/* $PREFIX/lib/
+#elif [[ ${target_platform} == linux-aarch64 ]]; then
+#    # Extract the .zip file
+#    mkdir -p tmp_extract
+#    unzip instantclient-basic-linux.arm64-$PKG_VERSION.zip -d tmp_extract/
+#
+#    # Copy contents to target lib folder
+#    cp -r tmp_extract/${OIC_FOLDER_NAME}/* $PREFIX/lib/
+#fi
+
+if [[ ${target_platform} == linux-* ]]; then
+    # Patch the RPATHs
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libnnz.so
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.10.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.11.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.12.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.18.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.19.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.20.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.21.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.22.1
+    patchelf --set-rpath '$ORIGIN/../lib' $PREFIX/lib/libclntsh.so.23.1
```

Comment:
```suggestion
    find ${PREFIX}/lib -iname "libclntsh.so*" -type f -exec patchelf --set-rpath '$ORIGIN/../lib' {} \;
```

Might make this a bit less tedious to read but also less explicit about what it's changing from a static code point-of-view.